pybamm/models/lead_acid/full.py

```diff
@@ -0,0 +1,83 @@
+#
+# Lead-acid full (1D) model
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+# import pybamm
```

Comment:
should this (empty) file be here?

Reply:
nope, just got lazy with branching, removing